charts/linkerd-crds/templates/policy/httproute.yaml

```diff
@@ -2,9 +2,12 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-
-  creationTimestamp: null
   name: httproutes.policy.linkerd.io
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    linkerd.io/control-plane-ns: {{.Release.Namespace}}
```

Comment:
The control-plane-ns annotation doesn't really make sense to use with CRDs (here or elsewhere). There are no guarantees or expectations that `.Release.Namespace` matches the linkerd control plane namespace.

Reply:
nevermind, this is used by `linkerd uninstall` currently